### DISCUSSION TITLE
user-guide/take-notes

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#82] [文档评论区]
# user-guide/take-notes

Zotero 支持直接打开并批注 PDF，与内部的笔记、同步等功能深度集成。建议使用 Zotero 阅读器以获得最佳体验。 内部阅读器所做的高亮、批注等不在原始 PDF 上，因此与第三方 PDF 阅读器的批注无法互通，也不支持非 Zotero 或非 webdav 的同步方式（关于同步，请见后续章节）。 如果你并没有特别依赖第三方 PDF 阅读器的某些特...

https://zotero-chinese.com/user-guide/take-notes.html

**Comment by 1102liu** [comment]
我想问一下使用webdav同步的话注释没有同步是因为什么啊

**Reply by northword** [reply]
注释由官方服务同步，可以被正常同步。你用的什么客户端？安卓端暂时无法同步注释，ios和pc都可以正常同步。

**Reply by 1102liu** [reply]
我现在搞的比较复杂，我用alist挂载阿里云盘，然后用alist将其映射为webdav，但是由于我没有公网ip，所以我没办法在两个电脑上登录同一个alist账户，所以我在两个电脑上分别创建了alist账户，挂载同一个阿里云盘，zotero账号也是不同电脑不同，这样子我两个zotero账户就是用的同一个webdav账户同步，但是这样子我发现好像不太行，不知道大佬对alist挂载云盘实现webdav功能有了解吗

**Reply by 1102liu** [reply]
就好像我在一个电脑上可以通过zotero的同步功能将文件保存在阿里云盘了，但是我第二个电脑上好像没办法把存的东西down下来

**Reply by 1102liu** [reply]
还有就是我看同步之后上传到云盘的是一个prop文件还有一个zip文件，大佬你的意思是这两个文件其实已经包含了注释了吗

**Reply by northword** [reply]
注释是通过官方服务同步的，和webdav无关。你需要使用同一个zotero账户，在所有设备上登录相同的zotero账号。

**Reply by 1102liu** [reply]
那我同一个账户不同的设备使用的webdav同步不同可以吗

**Reply by northword** [reply]
只要webdav里的内容一致即可

**Reply by 1102liu** [reply]
谢谢大佬，我明天尝试一下

**Comment by TrillStones** [comment]
我觉得注解做在pdf上，才方便带上所有pdf跑路啊。

**Reply by northword** [reply]
zotero出于笔记管理考虑将注释存储在数据库，但不限制注释导出。可以导出带注释的pdf。可以设置从外部pdf阅读器打开pdf以始终使新注释保留在pdf。

**Reply by panhaoyu** [reply]
放在pdf里面是无法进行后续的标签以及管理等操作的，会很不方便修改。

**Comment by worthwz** [comment]
我想请问一下，取消勾选拼接模式之后，还是会处在拼接模式是为什么？如何解决？

**Comment by Uangxin** [comment]
不支持画笔，感觉差点意思

**Reply by iZ1012** [reply]
我是二合一用户,非常想要直接在zotero的pdf阅读器里直接用笔画做笔记...

**Reply by northword** [reply]
Zotero 7 的是已经支持了的，预计今年下半年就会正式发布。如果想要体验测试版（不太推荐，普通用户还是用稳定版），可以在本文档-安装-小节里面找到下载地址。

**Comment by zhu-zhiyun** [comment]
请问，Zotero PDF支持类似于WPS PDF的文字批注或指示批注的功能吗？
或者让批注的文字一直显示在PDF页面上，不需要点击注释的图标才显示内容。
有什么插件可以实现这样的功能吗？

**Reply by Eva-Z7787** [reply]
超级想知道有没有类似插件！！

**Reply by panhaoyu** [reply]
有的吧，虽然我不知道是哪个插件提供的。

![image](https://github.com/user-attachments/assets/323d646d-483e-4fac-95db-cbbb1d46ba1f)

**Reply by northword** [reply]
这是 Zotero 原生提供的，现在已经支持文本注释了。

**Comment by Niky770** [comment]
你好！我想问一下我在主界面选中文献后（双击），并没有进入文献阅读模式，而是直接跳转到了知网里这篇文献的页面，请问这个怎么解决呀？非常感谢！

**Reply by iZ1012** [reply]
因为你点的条目只储存了它的网页快照，没有pdf附件。或者存了pdf但是优先打卡的是网页快照。总之至少有一个pdf附件，你才能看这篇文献呀。

**Comment by Niky770** [comment]
非常感谢！！我理解啦！！谢谢你的解答！🌹



---Original---
From: ***@***.***&gt;
Date: Wed, Aug 14, 2024 08:01 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [zotero-chinese/wiki] user-guide/take-notes (Discussion #82)




 
因为你点的条目只储存了它的网页快照，没有pdf附件。或者存了pdf但是优先打卡的是网页快照。总之至少有一个pdf附件，你才能看这篇文献呀。

**Comment by RonghaiHe** [comment]
能否将一个pdf中的高亮、批注转移到另一个相似的pdf上，因为有时候阅读了arxiv的文章，做了高亮、批注，后续作者更新了之后需要重新做高亮，不大方便

**Reply by northword** [reply]
据我所知目前不能。

对于 arxiv 工作流，可以参考 https://github.com/AllanChain/zotero-arxiv-workflow 插件。注意，我不确定这个插件是否可以实施你的需求，如果不能，你也许可以向插件作者提出一个功能请求。

一般来说，由于不同 pdf 版本可能存在排版和内容差异，因此我认为迁移批注不是一个很简单的事情，可能不太好实现。

**Comment by sanecho0502** [comment]
你好，我更新7之后，标注和高亮在电脑上都不显示，但是我看iPad平板端已经出现了，请问这是什么原因怎么解决呀

**Reply by sanecho0502** [reply]
我找到原因了！弄回来了！

**Reply by Zzea0224** [reply]
请问是什么原因呢，我也出现了同样的问题

**Comment by iampearl918** [comment]
请问Zotero “note” 笔记中，如何更改引用文献的样式呢？首次插入时跳出来“选择样式”的窗口（类似于在word中插入引文时），之后就不再出现了，想更改样式找不到地方！😭
> 截屏2024-09-03 16.37.15.png

**Reply by iampearl918** [reply]
(图片不知为何上传不了了……)

**Reply by iampearl918** [reply]
本来截了个图，显示“note”上方有一排工具按钮，有个“插入引注”的选项。我就是指的这个操作中出现的

**Comment by sunny-zhe** [comment]
为啥我每次笔记操作都会产生一堆未分类条目

**Reply by sunny-zhe** [reply]
重装软件解决了

**Comment by pyrojewel14** [comment]
使用手写笔记的时候相邻的笔迹不能被识别成同一个，导致左边笔记栏多出很多个独立的笔记词条，请问一下有合并手写笔记的功能吗

**Comment by MoQiii** [comment]
你好，我想请问一下在文献阅读的界面点击上的添加笔记（折了一角的正方形图标），然后点击文档选择某个位置会显示一个黄色折了一角的图标，内容也会显示在旁边，今天突然发现内容突然不显示在图标旁边，只显示在页面左边的预览区域，请问如何调整回去呢

**Comment by oojuzi** [comment]
请问文字注释怎么修改默认字体大小呢？现在默认大小为14，每次都需要手动调小

**Comment by aucu** [comment]
【求增加注释小功能】非常希望能在Zotero自带阅读器增加一个“区域高亮”功能，有时候一段话，要整个区域都高亮，而不是现在只能按照文章排版的句子高亮（句子中间毕竟还有空白间隔）。现在有可以选中区域的功能，但只是标注一个框，而没有高亮。 恳请把 高亮+区域选择 这两个结合起来，就形成了我提议的“区域高亮”（这是免费的PDFGear阅读器里的温馨功能，供参考）
